vocs.config.tsx

```diff
@@ -84,21 +77,37 @@ export default defineConfig({
       collapsed: false,
       items: [
         {
-          text: "Build a chat inbox",
-          link: "/inboxes/build-inbox",
+          text: "Pick an SDK",
+          link: "/inboxes/pick-an-sdk",
         },
         {
-          text: "Send disappearing messages",
-          link: "/inboxes/disappearing-messages",
+          text: "Create an EOA or SCW signer",
+          link: "/inboxes/create-a-signer",
         },
         {
-          text: "Manage inboxes",
-          link: "/inboxes/manage-inboxes",
+          text: "Create a client",
+          link: "/inboxes/create-a-client",
+        },
+        {
+          text: "Create conversations",
+          link: "/inboxes/create-conversations",
+        },
+        {
+          text: "Send messages",
+          link: "/inboxes/send-messages",
+        },
+        {
+          text: "List, stream, and sync",
+          link: "/inboxes/list-stream-and-sync",
         },
         {
           text: "Enable history sync",
```

Comment:
WDYT of changing this to just `History sync`? since it's enabled by default, the title implies that it needs to be enabled.